# VDBFUSION
voxel_size: 0.05
sdf_trunc: 0.1
space_carving: False

# Reconstruction
fill_holes: True
min_weight: 3.0

apply_pose: True
preprocess: True
min_range: 1.0
max_range: 75.0

pcl_topic: "/kitti/velo/pointcloud"

# TF
use_tf_transforms: true
parent_frame: "world"
child_frame: "base_link"
timestamp_tolerance_ns: 10000000
